INSTALL.md

```diff
@@ -81,13 +85,13 @@ The `docker run` command will mount your personal Documents folder to `/root/Doc
 can access your files.
 
 ```bash
-docker pull pycam/pycam  # downloads the latest image
-IP='<your local ip address>'
+docker build -t pycam/pycam .
+export IP='<your local ip address>'
```

Comment:
btw: do you require manual changes here?
Isn't "localhost" sufficient?

Reply:
Unfortunately, it does require manual intervention since the IP of the host is passed onto the docker image as an environment variable in the proceeding step.

This is all for X11 forwarding; it's routed from the virtualized docker image into the host, and so localhost wouldn't work. I don't believe there's a better alternative here but I can certainly be wrong.